new-evaluation.css

```diff
@@ -0,0 +1,143 @@
+		html {
+			height: 100%;
+			box-sizing: border-box;
+		}
+
+		body {
+			min-height: 100%;
+			position: relative;
+		
+		}
+		
+		.header {
+			height: 70px;
+			border-bottom: 1px solid grey;
+		}
+
+		#logo {
+			width: auto;
+			height: 50px;
+			line-height: 70px;
+			vertical-align: middle;
+		}
+
+		.div_form {
+			padding-top: 55px;
+			margin: 0 auto ;
+			width: 1000px;
+			height: 40px;
+		}
+
+		.input {
+			border: none;
+			border-bottom: 3px solid #f7f7f7;
+			height: 30px;
+			width: 300px;
+			margin: 7px;
+		}
+
+		.table {
+			border-collapse: collapse;
+			width: 1000px;
+			margin: 20.750px 0px;
+			text-align: center;
+		}
+		
+		th {
+			background-color: #419fe2;
+			height: 40px;
+			color: white;
+			font-family: Arial;
+			font-size: 14px;
+			width: 935px;
+			
+		}
+
+		.tr {
+			border: 1px solid black;
+			height: 40px;
+			width: 934px;
+		}
+
+		a {
+			margin-left: 100px;
+			margin-right: 15px;
+			font: 25px Arial;
```

Comment:
What do you think of setting the `Arial` font family to the body? Would the children inherit it?